engines/kgs-bot/README.md

```diff
@@ -0,0 +1,18 @@
+To run a bot on KGS, first build and start cpp-kgs container:
+
+`docker build . -f compose/cpp-kgs/Dockerfile -t humancompatibleai/goattack:cpp-kgs`
+
+`docker run --gpus all -v ~/go_attack:/go_attack -it humancompatibleai/goattack:cpp-kgs`
+
+Then activate the connection to KGS:
+
+`cd go_attack/engines/kgs-bot/`
+
+`java -jar kgsGtp.jar config.txt name=USERNAME password=PASSWORD`
```

Comment:
Minor issue, but it feels more Docker-style to make this the *entrypoint* to the Dockerfile, so you'd run it like:

`docker run --gpus all -v ~/go_attack:/go_attack -it humancompatibleai/goattack:cpp-kgs name=USERNAME password=PASSWORD`

Reply:
OK, I changed README to do it similar to this